src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -60,7 +84,12 @@ public Double averageSumOdd(List<Integer> numbers) {
      * Задача: Выбрать мужчин-военнообязанных (от `fromAge` до `toAge` лет)</p>
      **/
     public List<People> manSelectByAge(List<People> peopleList, int fromAge, int toAge) {
-        return Collections.emptyList();
+        return peopleList
+                .stream()
+                .filter(person -> person.getSex() == People.Sex.MAN)
+                .filter(person -> person.getAge() < toAge)
+                .filter(person -> person.getAge() > fromAge)
```

Comment:
same here - include edge cases as well
```suggestion
                .filter(person -> person.getAge() <= toAge)
                .filter(person -> person.getAge() >= fromAge)
```